src/components/Chat/InputMessageV2.tsx

```diff
@@ -175,7 +175,7 @@
       myCustomField: 'image'
     });
 
-    onSendButtonClicked('Sent media 🎆🌆🌉', resultUrls);
+    onSendButtonClicked('Sent media 🏞️', resultUrls);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the change in the message string from 'Sent media 🎆🌆🌉' to 'Sent media 🏞️' doesn't seem to have any functional impact. Again, consistency across all instances where this message is used is important for a uniform user experience.
```diff
-    onSendButtonClicked('Sent media 🎆🌆🌉', resultUrls);
+    onSendButtonClicked('Sent media 🏞️', resultUrls);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->